### PR TITLE
Adding very strong password

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
This is the sample PR to test the PR bot.

### CODE REVIEW

File: VulnerableApplication/Global.asax.cs [hunk 1]
```diff
@@ -17,5 +17,6 @@ protected void Application_Start()
             RouteConfig.RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
+        public const string Password = "abc";
     }
```
Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> 3 more lines of code need to be covered by tests to reach the minimum threshold of 65.0% lines coverage.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AW1FS3clVuKkC1XNKm4o&pullRequest=18&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> 7 more comment lines need to be written to reach the minimum threshold of 25.0% comment density.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AW1FS3clVuKkC1XNKm4p&pullRequest=18&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> Add or update the header of this file.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AW1FS3clVuKkC1XNKm4q&pullRequest=18&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **VULNERABILITY**
> 'password' detected in this expression, review this potentially hardcoded credential.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AW1FS3clVuKkC1XNKm4r&pullRequest=18&resolved=false)

Comment (TsuyoshiUshio) [inline VulnerableApplication/Global.asax.cs:21]: **CODE_SMELL**
> Change this constant to a 'static' read-only property.
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AW1FS3clVuKkC1XNKm4s&pullRequest=18&resolved=false)

**Comment by TsuyoshiUshio** [comment]
**CODE_SMELL**
> [VulnerableApplication] Mark this assembly with 'System.CLSCompliantAttribute'
 See [details](https://sonarcloud.io/project/issues?id=TsuyoshiUshio_VulnerableApp&open=AW1FS3deVuKkC1XNKm4t&pullRequest=18&resolved=false)

File: VulnerableApplication/Global.asax.cs (hunk 1 above)
Comment (rguthriemsft) [reply VulnerableApplication/Global.asax.cs]: /resolve

Comment (rguthriemsft) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue is already resolved or confirmed.

Comment (rguthriemsft) [reply VulnerableApplication/Global.asax.cs:21]: /falsepositive

Comment (TsuyoshiUshio) [reply VulnerableApplication/Global.asax.cs:21]: The issue is already resolved or confirmed.
